### PR TITLE
Merging builder-centos into builder-fedora

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

**Comment by fepitre** [comment]
Let Travis build all things.

**Comment by fepitre** [comment]
Only CentOS for R3.2 is not handle. Should I remove it or you plan to build the template for R3.2 as there is several users for it?

**Comment by fepitre** [comment]
I prepared the builder and builder-fedora on qubes-builder there is a branch named buider-yum
‎and on builder-fedora there is a branch of builder-rpm2(current one here) named builder-yum. Maybe we could do the merge and the change of name directly?

**Comment by marmarek** [comment]
> Only CentOS for R3.2 is not handle. Should I remove it or you plan to build the template for R3.2 as there is several users for it?

I don't mind building it for R3.2 too. Have you tested the build?

**Comment by fepitre** [comment]
I'm daily using Centos. Now it's in r4 but also in r3.2 before. With respect to this commit I have not tested the build but I can. It should be fine.

**Comment by marmarek** [comment]
> I prepared the builder and builder-fedora on qubes-builder there is a branch named builder-yum
> ‎and on builder-fedora there is a branch of builder-rpm2(current one here) named builder-yum. Maybe we could do the merge and the change of name directly?

Renaming components is a pain, so I'd like to avoid it unless absolutely necessary. Having builder-fedora being also a plugin for CentOS indeed may be confusing, so it may be a good reason for renaming. I need to think about it.

**Comment by marmarek** [comment]
As for CentOS template for R3.2 - I need to generate signing key for it (currently there is only one for R4.0)

File: prepare-chroot-builder [hunk 1]
```diff
@@ -108,7 +108,12 @@ cp ${PLUGIN_DIR}/repos/builder-local.repo $DIR/etc/yum.repos.d/
 sed -i -e "s#ROOT#$PWD#" $DIR/etc/yum.repos.d/*-local.repo
 
 if [ -n "$USE_QUBES_REPO_VERSION" ]; then
-    cp ${PLUGIN_DIR}/repos/qubes-repo-vm.repo $DIR/etc/yum.repos.d/
```
Comment (marmarek) [inline prepare-chroot-builder]: `qubes-repo-vm-${DISTRIBUTION}.repo`?

Comment (fepitre) [reply prepare-chroot-builder]: Yes good point.

File: prepare-chroot-builder [hunk 2]
```diff
@@ -1,57 +1,65 @@
...
 PLUGIN_DIR="`dirname $0`"
 
 DIR=$1
-DISTRO=$2
+DIST=$2
 CHROOT_CACHE_FILE=
+
+if grep -q fc <<< "$DIST"; then
+    YUM=dnf
+    DIST_NAME="fedora"
```
[the remark this answers is not in the document]
Comment (marmarek) [reply prepare-chroot-builder]: What about this one? (and its usage later in this script)

**Pull request merged**